js/views/modals/purchase/Purchase.js

```diff
@@ -452,6 +452,7 @@ export default class extends BaseModal {
         singleSelect: true,
         selectFirst: true,
         radioStyle: true,
+        useCache: false,
```

Comment:
If we don't want to usecache, then that's what's in place now, because if you don't pass it in, it defaults to false.